docs/json.md

```diff
@@ -121,3 +121,36 @@ HEMMT will build the specified optionals from the `./optionals` folder.
     "particles"
 ]
 ```
+
+## headerexts
+**Type**: Array \[String\]
+
+HEMTT will apply specified header extensions to each PBO.
+
+```json
+"headerexts": [
+    "author=me"
```

Comment:
We should probably be doing author ourselves since we already store the author in the project file.

Reply:
No, this is a bad example maybe, but we shouldn't store anything ourselves in there, all it adds is PBO size. We can support author with a replacement pattern though. 

ACE/CBA store git hash here for issue reporting purposes (gets displayed in RPT), that's what I will use it for primarily, but git hash part has yet to be done.

Reply:
Btw, those docs will get updated with possible replacement patterns when I add git hash utility. Example will be better then!